pkg/virt-handler/vm_test.go

```diff
@@ -1465,66 +1469,136 @@ var _ = Describe("VirtualMachineInstance", func() {
 
 	})
 	Context("When VirtualMachineInstance is connected to a network", func() {
-		It("should report the status of this network", func() {
-			vmi := NewPendingVirtualMachine("testvmi")
-			vmi.Status.Phase = v1.Scheduling
-			pod := NewPodForVirtualMachine(vmi, k8sv1.PodRunning)
 
-			addVirtualMachine(vmi)
-			podFeeder.Add(pod)
+		It("should only report the pod network in status", func() {
 
-			networkName := "test net"
-			podIp := "1.1.1.1"
-			pod.Status.PodIP = podIp
+			vmi := v1.NewMinimalVMI("testvmi")
+			vmi.UID = vmiTestUUID
+			vmi.ObjectMeta.ResourceVersion = "1"
+			interface_name := "interface_name"
 			vmi.Spec.Networks = []v1.Network{
-				v1.Network{
-					Name: networkName,
-					NetworkSource: v1.NetworkSource{
-						Pod: &v1.PodNetwork{
-							VMNetworkCIDR: "1.1.1.1",
-						},
-					},
+				{
```

Comment:
Since the test name is "should only report the pod network in status" I think you should have here more than one network (like the [original](https://github.com/kubevirt/kubevirt/blob/00dde8579deea1366b11b58469b955744ea2945b/pkg/virt-controller/watch/vmi_test.go#L1058) virt-controller test had).

Reply:
added